pydeseq2/dds.py

```diff
@@ -281,9 +286,13 @@ def __init__(
         self.min_replicates = min_replicates
         self.beta_tol = beta_tol
         self.quiet = quiet
+        self.low_memory = low_memory
         self.logmeans = None
         self.filtered_genes = None
 
+        # Cooks outliers
+        self._cooks_outlier = None
```

Comment:
Following the anndata logic, It seems like it would be more natural to store this in `.varm` rather than as a `DeseqDataSet` attribute on its own.

I would also suggest renaming to `_pvalue_cooks_outlier` to disambiguate between genes whose counts were replaced based on cook's distances (`.varm['replaced']`) and those whose p-values are set to NaN later on (again based on cook's).

Reply:
Moved the outliers mask into `self.varm['_pvalue_cooks_outlier']`